app/assets/stylesheets/includes/forms.scss

```diff
@@ -36,6 +36,20 @@ section.sign-up {
   .btn-link {
     color: $white
   }
+  h1, h2, h3, h4, h5 {
```

Comment:
I would prefer to change this to something like `.social-login-title` and add it to the html element.